ref-exchange/src/account_deposit.rs

```diff
@@ -90,24 +173,35 @@ impl AccountDeposit {
 
 #[near_bindgen]
 impl Contract {
+    /// Deposits attached NEAR into predecessor account deposits. The deposited near will be used
+    /// for trades and for storage. Predecessor account must be registered. Panics otherwise.
+    /// NOTE: this is a simplified and more direct version of `storage_deposit` function.
+    #[payable]
+    pub fn deposit_near(&mut self) {
+        let sender_id = env::predecessor_account_id();
+        let mut acc = self.get_account(&sender_id);
```

Comment:
should we register here if not registered? it seems like this will be easier to use than storage_deposit in general case.

Reply:
I was thinking about it. We have a standard for user / account registration. This function is to make it more clear to deposit more NEAR and not confuse with registration. However if you think that by allowing a registration here, we will improve UX then we can do that.